frontend/src/worksheets/TableListOfWorkSheets.js

```diff
@@ -23,6 +24,11 @@ export default function TableListOfWorkSheets() {
     setStatus: onStatus,
   } = useWorkSheets()
 
+  async function handlePrint(worksheet) {
+    const partnerData = await getPartner(worksheet.partnerId)
+    createWorksheetPDF(worksheet, partnerData.data)
```

Comment:
Milyen jó lenne, ha az API réteg már kihámozná a data-t és csak a partner adatot adná vissza nem az egész Axios response-t...

Reply:
Ez a többi API hívásra is érvényes, csak eddig nem tűnt föl.